.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "oni-lda-c"]
-	path = oni-lda-c
-	url = https://github.com/OpenNetworkInsight/oni-lda-c
```

Comment:
dammit
this was not supposed to be removed

not sure why the file got blown away